contracts/universalSchemes/DaoCreator.sol

```diff
@@ -198,16 +200,19 @@ contract DaoCreator {
         // Create Controller:
         if (UController(0) == _uController) {
             controller = ControllerInterface(controllerCreator.create(avatar));
-            avatar.transferOwnership(address(controller));
-            // Transfer ownership:
-            nativeToken.transferOwnership(address(controller));
-            nativeReputation.transferOwnership(address(controller));
         } else {
             controller = _uController;
-            avatar.transferOwnership(address(controller));
-            // Transfer ownership:
-            nativeToken.transferOwnership(address(controller));
-            nativeReputation.transferOwnership(address(controller));
+        }
+
+        // Add the DAO to the tracking registry
+        daoTracker.track(avatar, controller);
```

Comment:
how about moving that after transfer ownership to the controller and pass to daoTracker.track only the avatar ?

Reply:
See response for question about removing the `_controller` parameter. The conclusion there dictates whether this change is possible.